product_docs/docs/migration_portal/4.0/04_mp_migrating_database/03_mp_schema_migration.mdx

```diff
@@ -175,36 +197,37 @@ Migrate schemas to an on-premises EDB Postgres Advanced Server on Linux.
 
```

Comment:
In the command examples above, "<project_name>.sql" should be changed to something like "<sql_file_name>.sql" since there are more import file options now available.

Reply:
Fixed.